pdfmaker/css/holt/pdf.css

```diff
@@ -482,40 +532,54 @@ nav[data-type="toc"] h1 {
 nav[data-type="toc"] ol {
   margin-left: 0; }
 
-nav[data-type="toc"] li {
+nav[data-type="toc"] li, section.texttoc p {
   font-size: 11.2pt;
   line-height: 15pt;
   margin-bottom: 6pt;
   margin-left: 0;
   list-style-type: none; }
 
+section.texttoc p {
+  margin-top: 0;
+  text-indent: 0; }
+
 nav[data-type="toc"] li.PartNumberpn ~ li,
-nav[data-type="toc"] li.PartTitlept ~ li {
-  margin-bottom: 0; }
+nav[data-type="toc"] li.PartTitlept ~ li,
+section.texttoc p.PartTitlept ~ p,
+section.texttoc p.PartNumberpn ~ p {
+  margin-bottom: 0;
+  margin-top: 0; }
 
 nav[data-type="toc"] li.PartNumberpn ~ li.PartNumberpn,
 nav[data-type="toc"] li.PartTitlept ~ li.PartTitlept,
 nav[data-type="toc"] li.PartNumberpn ~ li.PartTitlept,
-nav[data-type="toc"] li.PartTitlept ~ li.PartNumberpn {
+nav[data-type="toc"] li.PartTitlept ~ li.PartNumberpn,
+section.texttoc p.PartNumberpn ~ p.PartNumberpn,
+section.texttoc p.PartTitlept ~ p.PartTitlept,
+section.texttoc p.PartNumberpn ~ p.PartTitlept,
+section.texttoc p.PartTitlept ~ p.PartNumberpn {
   margin-bottom: 8.44pt; }
 
-nav[data-type="toc"] li.PartNumberpn + li.PartTitlept {
+nav[data-type="toc"] li.PartNumberpn + li.PartTitlept,
+section.texttoc p.PartNumberpn + p.PartTitlept {
   margin-top: -8.44pt; }
 
 nav[data-type="toc"] a {
   text-decoration: none;
   color: cmyk(0%, 0%, 0%, 100%); }
 
 nav[data-type="toc"] li a:after {
-  content: target-counter(attr(href), page);
-  float: right; }
+  content: leader(space) target-counter(attr(href), page);
+  padding-left: 15pt }
```

Comment:
You don't need to change this now, but our standard practice is to end all lines with a closing ";", even if it's the last line in a definition. Also, you should make sure to use spaces consistently (it's just cleaner to read--see line 577 below).